src/js/videojs-dash.js

```diff
@@ -73,11 +73,18 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
-    // Apply any options that are set
-    if (options.dash && options.dash.limitBitrateByPortal) {
```

Comment:
We should still include this code with a deprecation warning and remove it in the next major version.